.dockerignore

```diff
@@ -1,3 +1,10 @@
 .git
 .github
+.gitignore
 task-definition*
+**/target
+docker-compose.yml
+LICENSE
+Dockerfile
+task-definition*
+README*
```

Comment:
Why do you ignore `.gitignore`, `LICENSE`, `README` and the `task-definitions`?

Reply:
I tried to exclude anything that's not build-relevant; why should they be copied to the container if they're useless there?

Reply:
I am no license expert, but maybe we should include a license with our container? Not sure if it is safe to remote it.

Reply:
sounds reasonable. I'll put LICENSE and README back in